src/main/java/com/snowflake/kafka/connector/records/SnowflakeAvroConverter.java

```diff
@@ -168,8 +168,7 @@ public SchemaAndValue toConnectData(final String s, final byte[] bytes) {
               id));
     } catch (Exception e) {
       if (breakOnSchemaRegistryError) {
-        throw SnowflakeErrors.ERROR_0010.getException(
-                "Failed to parse AVRO " + "record\n" + e.toString());
+        throw SnowflakeErrors.ERROR_0010.getException(e);
```

Comment:
should we keep the error message?

Reply:
The original implementation dropped further meta data about the exception e, e.g., the stack strace. As I thought the exceptions from the Avro implementation are informative enough I decided to use the getException(...) implementation that just wraps the original exception. I see that this is a trade-off, and I am completely fine with reverting here.

Reply:
I have reverted to the original error message now.